### ISSUE TITLE
aws_apigatewayv2_stage does not output execution_arn

### ISSUE THREAD

**Issue by eimantas-gecas-axomic** [terraform-providers/terraform-provider-aws#14554]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version
Terraform v0.12.23
+ provider.aws v3.1.0

### Affected Resource(s)

* aws_apigatewayv2_stage

### Terraform Configuration Files

```hcl
resource aws_apigatewayv2_api this {
  name = "test_api"
  protocol_type = "HTTP"
}

resource aws_apigatewayv2_stage default {
  api_id = aws_apigatewayv2_api.this.id
  name = "$default"
  auto_deploy = true
}
```

### Expected Behavior

`aws_apigatewayv2_stage.default.execution_arn` should equal to something similar to `arn:aws:execute-api:us-east-1:538692352:g34t324gr/*/$default`

### Actual Behavior

`aws_apigatewayv2_stage.default.execution_arn`  is empty

**Comment by ewbankkit** [comment]
@eimantas-gecas-axomic Thanks for raising this issue.
Indeed, this seems to be an oversight as both the [original blog post](https://aws.amazon.com/blogs/compute/announcing-http-apis-for-amazon-api-gateway/)

![blog-post-execute-arn](https://user-images.githubusercontent.com/2404182/89899020-aad3fa00-dbaf-11ea-9fd0-04fb3157a2b5.PNG)

and the relevant [Troubleshooting section of the Developer Guide](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-troubleshooting-lambda.html)

![developer-guide-execute-arn](https://user-images.githubusercontent.com/2404182/89898869-6ea09980-dbaf-11ea-933b-676b3f9fe808.PNG)

mention adding a Lambda resource policy permission with the `execution_arn` value as the source ARN.
I'll address this.

**Comment by ewbankkit** [comment]
As the [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/arn-format-reference.html#apigateway-execute-api-arns) states that the REST and HTTP `execute-api` ARN formats are the same we'll output the `execution_arn` attribute in the same format as the [`aws_api_gateway_stage`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_stage) resource's equivalent:

> arn:_partition_:execute-api:_region_:_account-id_:_api-id_/_stage_

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 3.3.0 of the Terraform AWS Provider, next week. Thanks to @ewbankkit for the fix. 👍
